lib/ansible/utils/collection_loader/_collection_finder.py

```diff
@@ -43,6 +38,11 @@ def import_module(name):
     # 2.7 has a global reload function instead...
     reload_module = reload  # pylint:disable=undefined-variable
 
+try:
+    from importlib.util import spec_from_loader
+except ImportError:
+    pass
```

Comment:
It shouldn't ever happen, but maybe I should set spec_from_loader to None here, and raise a ValueError in find_spec if it's None.